### PR TITLE
Implementing ArrayList in Java

### LINKED ISSUE #1889: Implementing Collections in Java
I would like to implement collections (List, Stack, Queue and all the implementations of these interfaces) in Java. Please assign me this issue.

### PR DESCRIPTION
**Pull request by simrankoul**
Fixes #1889 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of the file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Collections/arrayList.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import java.util.*;
+public class arrayList {
+    public static void main(String[] args) {
+       
+        //Creating the ArrayList
+        List<Integer> li = new ArrayList<Integer>();
+        
+        //Adding Objects to the List
+        li.add(1);
```
Comment (jainaman224) [inline Collections/arrayList.java]: Use dynamic input.

File: Collections/arrayList.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        
+        //To check whether the list is empty or not.
+        System.out.println("Is the list empty? "+li.isEmpty());
+        
+        //To clear all elements in array
+        li.clear();
+        
+        
+    }
+}
```
Comment (jainaman224) [inline Collections/arrayList.java]: Add sample input output.

File: Collections/arrayList.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import java.util.*;
+public class arrayList {
+    public static void main(String[] args) {
+       
+        //Creating the ArrayList
```
Comment (jainaman224) [inline Collections/arrayList.java]: Add space after `//`.

File: Collections/arrayList.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import java.util.*;
+public class arrayList {
```
Comment (jainaman224) [inline Collections/arrayList.java]: Add new line above this line.

File: Collections/arrayList.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
+import java.util.*;
```
Comment (jainaman224) [inline Collections/arrayList.java]: Make this program menu driven.

**Comment by jainaman224** [comment]
@simrankoul Are you working on it?

**Comment by simrankoul** [comment]
@jainaman224 yes I am. I will get it done by tomorrow.

**Comment by simrankoul** [comment]
@jainaman224  Kindly check if I have done the required modifications correctly so that I can also submit for the other Collections as well.

File: Collections/arrayList.java [hunk 6]
```diff
@@ -0,0 +1,228 @@
+
```
Comment (raghav-dalmia) [inline Collections/arrayList.java]: Remove this line and add a line before `arrayList` class

File: Collections/arrayList.java [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+                    break;
+                    
+            }
+            System.out.println("Do you want to continue? Enter 'y' for more inputs or 'n' to stop");
+            choice=scan.next().charAt(0);
+        }
+    }
+}
+
+/* SAMPLE INPUT/OUTPUT
```
Comment (raghav-dalmia) [inline Collections/arrayList.java]: it should be
```
/*     
Sample Input ...
```

**Comment by raghav-dalmia** [comment]
Also, squash your commits.

**Pull request merged**